### DISCUSSION TITLE
Pin tab group, such that it doesn't disappear when its last remaining tab is closed?

### DISCUSSION THREAD

**Discussion by PANstudio** [mbnuqw/sidebery#1394] [Q&A] [answered]
![image](https://github.com/mbnuqw/sidebery/assets/4048196/e49fa22b-7765-4e7f-8fba-3aab9fc86cdb)

I have two tab groups - work and personal. I'd like these to always show, even if they have no tabs in. Right now, when I close the remaining tab in a group, the group disappears (that is, its icon (see red arrow) disappears.

This doesn't seem to delete the group; I can revisit it by selecting the group from the arrow dropdown that appears when a group is emptied.

![image](https://github.com/mbnuqw/sidebery/assets/4048196/9e8089b9-f8bb-4383-8fe1-a7181327ed21)

Is there a way to pin tab groups?

Thank you.

PS Apologies if there's an option for this - I couldn't see one, but there's lots of settings in Sidebery and I'm new to it.

**Comment by PANstudio** [comment, marked as answer]
Well I just sort of found my answer! I'll leave this discussion here in case it helps anyone else.

The solution is to right click on the tab group and choose "configure panel". Then look for the option "Create new tab after the last one is closed." This ensures that, when a group is emptied, a new tab is opened so it's not really empty. This persists the group.

Not perfect, perhaps - it's weird to have to have an empty tab sitting there just to persist the group, but it's a solution for now. Still think it would be good to have a "pin this group" option.

**Comment by albino1** [comment]
> PS Apologies if there's an option for this - I couldn't see one, but there's lots of settings in Sidebery and I'm new to it.

I'm pretty sure what you're looking for is just:

`Settings -> Navigation Bar -> Hide empty tabs panels -> off`
